src/elements/content-preview/ContentPreview.scss

```diff
@@ -41,7 +41,7 @@
         position: absolute;
         top: 50%;
         transform: translateY(-50%);
-        transition-property: opacity;
+        transition: opacity .5s, left .3s cubic-bezier(.4, 0, .2, 1);
```

Comment:
Where does the left value get changed?

Reply:
When thumbnails sidebar is opened: https://github.com/box/box-ui-elements/pull/1032/files/444bc2254419aba9b36fb8b4d76fe6208396c674#diff-c8b2e5e96e87b90de7160a27e8aa2805R84